src/System.Diagnostics.EventLog/pkg/System.Diagnostics.EventLog.pkgproj

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Diagnostics.EventLog.csproj">
+      <SupportedFramework>netcoreapp2.0</SupportedFramework>
```

Comment:
I think adding net461 to your list of supported frameworks will fix the validation error you are hitting.